package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "supertokens-auth-react",
-    "version": "0.42.2",
+    "version": "0.42.3",
```

Comment:
After some discussion with @rishabhpoddar, this should be a breaking change, so please:
- update the version numbers to 0.43.0 everywhere
- update the branch target

Reply:
Also, please update the size limit checks, since that check is still failing.

Reply:
Done ✅